test/integration/tasks.test.js

```diff
@@ -394,6 +394,65 @@ describe("Tasks", function () {
       expect(res).to.have.status(403);
       expect(res.body.message).to.be.equal("Status cannot be updated. Please contact admin.");
     });
+
+    it("should give a response message as 'Task updated but another task not found' if we have completed 100% task and a task is not available", async function () {
+      const taskData = {
+        title: "Test task",
+        type: "feature",
+        endsOn: 1234,
+        startedOn: 4567,
+        status: "IN_PROGRESS",
+        percentCompleted: 70,
+        participants: [],
+        assignee: appOwner.username,
+        completionAward: { [DINERO]: 3, [NEELAM]: 300 },
+        lossRate: { [DINERO]: 1 },
+        isNoteworthy: true,
+      };
```

Comment:
I tried that was not working, somehow it was only working if we put it here

Reply:
I believe this is because of `appOwner.username`.

fixture data is a function, so pass it via argument. Something like 

`const taskData = taskDataFixture(appOwner.username);`

Reply:
okay let me see